tests/Fixer/FunctionNotation/PhpdocToParamTypeFixerTest.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\FunctionNotation;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Jan Gantzert <jan@familie-gantzert.de>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\FunctionNotation\PhpdocToParamTypeFixer
+ */
+final class PhpdocToParamTypeFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     * @param null|int    $versionSpecificFix
+     * @param null|array  $config
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null, $versionSpecificFix = null, $config = null)
+    {
+        if (
+            (null !== $input && \PHP_VERSION_ID < 70000)
+            || (null !== $versionSpecificFix && \PHP_VERSION_ID < $versionSpecificFix)
+        ) {
+            $expected = $input;
+            $input = null;
+        }
+        if (null !== $config) {
+            $this->fixer->configure($config);
+        }
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return [
+            'no phpdoc param' => [
+                '<?php function my_foo() {}',
+            ],
+            'invalid param' => [
+                '<?php /** @param */ function my_foo() {}',
+            ],
+            'invalid class 1' => [
+                '<?php /** @param \9 */ function my_foo() {}',
+            ],
+            'invalid class 2' => [
+                '<?php /** @param \\Foo\\\\Bar */ function my_foo() {}',
+            ],
+            'blacklisted class methods' => [
+                '<?php
+
+                    class Foo
+                    {
+                        /** @param Bar */
+                        function __construct() {}
```

Comment:
Shouldn't you have the params, to be sure it's untouched? This case isn't doing anything anyway now because of doc/signature mismatch.